.azure-pipelines/release.yml

```diff
@@ -0,0 +1,85 @@
+trigger:
+  tags:
+    include:
+      - '*.*.*'
+
+pr: none
+
+stages:
+
+  - stage: 'build_conda_package'
+    displayName: 'Build Conda Package'
+    jobs:
+      - job: 'linux'
+        displayName: 'Linux'
+        timeoutInMinutes: 120
+        strategy:
+          matrix:
+            Python37:
+              PYTHON_VERSION: '3.7'
```

Comment:
Probably fine for now but we probably want to build other versions of Python as well in the future

Reply:
Yes, that's why I went with the Matrix rather than hard-coding. Should make it easier to add as needed.